core/src/main/java/brooklyn/event/feed/windows/WindowsPerformanceCounterFeed.java

```diff
@@ -251,10 +258,71 @@ public T call() throws Exception {
         }
     }
 
+    private static class SendPerfCountersToSensors implements PollHandler<WinRmToolResponse> {
+
+        private final EntityLocal entity;
+        private final List<WindowsPerformanceCounterPollConfig<?>> polls;
+
+        public SendPerfCountersToSensors(EntityLocal entity, List<WindowsPerformanceCounterPollConfig<?>> polls) {
+            this.entity = entity;
+            this.polls = polls;
+        }
+
+        @Override
+        public boolean checkSuccess(WinRmToolResponse val) {
+            if (val.getStatusCode() != 0) return false;
+            String stderr = val.getStdErr();
+            if (stderr == null || stderr.length() != 0) return false;
+            String out = val.getStdOut();
+            if (out == null || out.length() == 0) return false;
+            return true;
+        }
+
+        @Override
+        public void onSuccess(WinRmToolResponse val) {
+            String[] values = val.getStdOut().split("\r\n");
+            for (int i = 0; i < polls.size(); i++) {
+                Class<?> clazz = polls.get(i).getSensor().getType();
+                Maybe<? extends Object> maybeValue = TypeCoercions.tryCoerce(values[i], TypeToken.of(clazz));
```

Comment:
Does use of `tryCoerce` mean that we'll ignore it if the returned value was of an unexpected format? Would seem good to log at least once to say that (e.g. if get "1.2" when we expect an integer, then we should log that somewhere sensible).